designer/server/src/main/scala/pl/touk/nussknacker/ui/api/NodesApiEndpoints.scala

```diff
@@ -482,6 +482,7 @@ object NodesApiEndpoints {
       final case object NoPermission                                    extends NodesError with CustomAuthorizationError
       final case class MalformedTypingResult(msg: String)               extends NodesError
 
+      // FIXME: what does this message mean??
```

Comment:
Please, remove comment. I guess here we need one-way serialization only to communicate exceptions.